### PR TITLE
Cleanup Publication Name from source PostgreSQL DB

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
Jenkins Automation Test: https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/1678/

File: yb-voyager/cmd/endMigrationCommand.go [hunk 1]
```diff
@@ -381,6 +382,19 @@ func deletePGReplicationSlot(msr *metadb.MigrationStatusRecord, source *srcdb.So
 	}
 }
 
+func deletePGPublication(msr *metadb.MigrationStatusRecord, source *srcdb.Source) {
+	if msr.PGPublicationName == "" || source.DBType != POSTGRESQL {
+		log.Infof("pg publication name is not set or source db type is not postgresql. skipping deleting pg publication name")
```
Comment (makalaaneesh) [inline yb-voyager/cmd/endMigrationCommand.go:387]: nit: it'll be also valuable to have the log for the inverse case: i.e. when we are indeed deleting the publication name.

File: yb-voyager/src/dbzm/config.go [hunk 2]
```diff
@@ -269,8 +272,8 @@ func (c *Config) String() string {
 			c.SSLKey,
 			c.SSLRootCert)
 		conf = conf + sslConf
-		if c.ReplicationSlotName != "" {
-			conf = conf + fmt.Sprintf(postgresReplicationSlotNameTemplate, c.ReplicationSlotName)
+		if c.ReplicationSlotName != "" || c.PublicationName != "" {
```
Comment (makalaaneesh) [inline yb-voyager/src/dbzm/config.go:275]: The "OR" check seems rather confusing. (If either is set, we are setting both in the config)
Maybe separate out both the configs?

**Review by makalaaneesh — APPROVED** [review]
LGTM!

**Pull request merged**